tests/src/gui/testqgsvaluerelationwidgetwrapper.cpp

```diff
@@ -3,7 +3,9 @@
      --------------------------------------
     Date                 : 21 07 2017
     Copyright            : (C) 2017 Paul Blottiere
+    Copyright            : (C) 2018 Alessandro Pasotti
```

Comment:
Do we do this? I always thought it was just whoever created the file gets added, and everyone else just accepts that :)

Reply:
no problems to remove